app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -599,8 +599,8 @@ def hide_button?(id)
       type = get_vmdb_config.fetch_path(:server, :remote_console_type)
       return type != 'MKS' || !@record.console_supported?(type)
     when "vm_vnc_console"
-      type = get_vmdb_config.fetch_path(:server, :remote_console_type)
-      return type != 'VNC' || !@record.console_supported?(type)
+      # :remote_console_type is only useful for vmware consoles
+      return !@record.console_supported?('vnc')
```

Comment:
If the admin sets the console access for VMware VMs to VMRC or MKS we won't potentially show two buttons (admin's choice and VNC) that say VM Console per the changes in my original pull request.